source/eventcore/drivers/posix/sockets.d

```diff
@@ -24,6 +24,8 @@ version (Posix) {
 	version (linux) enum SO_REUSEPORT = 15;
 	else enum SO_REUSEPORT = 0x200;
 
+	version (linux) enum IP_TRANSPARENT = 19;
```

Comment:
```suggestion
	version (linux) {
		static if (!is(typeof(IP_TRANSPARENT)))
			enum IP_TRANSPARENT = 19;
	}
```

To avoid breaking in the future, if druntime adds the `IP_TRANSPARENT` declaration.